idiss-csharp/IdissLib/Class1.cs

```diff
@@ -0,0 +1,333 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace IdissLib
+{
+
+    /// Here we provide classes that match some of the Rust types that involved with identity validation and creation. 
+    /// Most of the field names of these classes are chosen so that they match JSON fields of JSON serialization of the corresponding Rust types.
+    public class YearMonth {
```

Comment:
Use consistent placement of `{ }`. In C# the usual convention is to always put a newline before, so it would be:
```
public Class YearMonth
{
...
}
```
In this class I see a mix of the styles. I haven't used C# for quite some time, but perhaps there is some standardized linter you could use (VS code probably does it for you too).